apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -742,6 +857,28 @@ static void tc_fs_vfs_seekdir(void)
 	itoa((int)offset, filename, 10);
 	TC_ASSERT_EQ("readdir", strncmp(dirent->d_name, filename, 1), 0);
 
+	/* For Negative offset in seekmountdir operations */
+	dirp = opendir(VFS_FOLDER_PATH);
+	TC_ASSERT_NEQ("opendir", dirp, NULL);
+
+	offset = -2;
+	seekdir(dirp, offset);
+	TC_ASSERT_NEQ_CLEANUP("seekdir", dirp, NULL, closedir(dirp));
+
+	/* for pseudo dir operations */
+
+	dirp = opendir(ROOT);
```

Comment:
The dirp is updated by this line. But you didn't close previous opened. Above cleanup is calling when TC_ASSERT meets failure. In success case, this is a leak.

Reply:
Okay. Will close that dirp.